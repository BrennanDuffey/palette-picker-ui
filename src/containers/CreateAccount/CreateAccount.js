import React, { Component } from 'react';

class CreateAccount extends Component {
  render() {
    return (
      <section className="CreateAccount">
        <form 
          action="post" 
          className="CreateAccount-form">
          <h2>Create an Account</h2>
          <label htmlFor="username-input">USERNAME:</label>
          <input 
            type="text" 
            id="username-input"
          />
          <label htmlFor="password-input-1">PASSWORD:</label>
          <input 
            type="password"
            id="password-input-1"
          />
          <label htmlFor="password-input-2">RE-ENTER PASSWORD:</label>
          <input 
            type="password"
            id="password-input-2"
          />
          <button className="new-account-btn">
            SUBMIT
          </button>
        </form>
      </section>
    );
  }
}

export default CreateAccount;