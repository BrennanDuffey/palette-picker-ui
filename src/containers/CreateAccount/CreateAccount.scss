.CreateAccount {
  background: $lightGray;
  padding: 50px 100px;

  form {
    background: white;
    display: flex;
    flex-direction: column;
    padding: 50px;

    h2 {
      font-size: 1.8em;
      margin: 0 0 20px;
    }

    label {
      font-size: .9em;
      margin: 5px 0;
    }

    input,
    button {
      border: solid 2px $lightGray;
      background: none;
      margin: 0 0 20px 0;
      padding: 12px;
    }

    input {
      font-size: 1em;
    }

    button {
      background: $gray;
      border: none;
      color: white;
      font-size: 1.2em;
      font-weight: 800;
      margin: 30px 0;

      &:hover {
        background: $lightGray;
      }
    }
  }
}