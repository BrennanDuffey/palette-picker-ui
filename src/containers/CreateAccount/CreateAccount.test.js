import React from 'react';
import { shallow } from 'enzyme';
import { CreateAccount } from './CreateAccount';

describe('CreateAccount', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <CreateAccount />
    );
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});