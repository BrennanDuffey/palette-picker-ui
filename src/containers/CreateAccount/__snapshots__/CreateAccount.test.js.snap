// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateAccount should match snapshot 1`] = `
<section
  className="CreateAccount"
>
  <form
    action="post"
    className="CreateAccount-form"
    onSubmit={[Function]}
  >
    <h2>
      Create an Account
    </h2>
    <label
      htmlFor="username-input"
    >
      USERNAME:
    </label>
    <input
      id="username-input"
      name="username"
      onChange={[Function]}
      type="text"
    />
    <label
      htmlFor="password-input-1"
    >
      PASSWORD:
    </label>
    <input
      id="password-input-1"
      name="password1"
      onChange={[Function]}
      type="password"
    />
    <label
      htmlFor="password-input-2"
    >
      RE-ENTER PASSWORD:
    </label>
    <input
      id="password-input-2"
      name="password2"
      onChange={[Function]}
      type="password"
    />
    <button
      className="new-account-btn"
    >
      SUBMIT
    </button>
  </form>
</section>
`;
